roles/arteria-delivery-ws/templates/arteria-delivery_wrapper.sh.j2

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash -l
+
+module load dds-cli
```

Comment:
```suggestion
module load bioinfo-tools dds-cli/{{ dds_cli_version }}
```

Reply:
I agree with @matrulda, for now I think it's better to always load the last version. We had this problem when they released DDS 2.0.0. When DDS is released on a more predictable basis we can start pinning versions then.

Reply:
Yes, that's true :+1: Good catch